apps/src/templates/teacherDashboard/RecentCourses.story.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import RecentCourses from './RecentCourses';
+import CourseCard from './CourseCard';
+
+const EXAMPLE_CARD_DATA = {
+  courseName: "CSP Unit 2 - Digital Information",
+  description: "Explore how more complex digital information is represented and manipulated through computation and visualization",
+  image: "this is where there will be the source for the photo",
+  link: "link to wherever you want the button to go...",
+  assignedSections: [] //<-- this info will be used in the tooltip,
+};
+
+const COURSES_DATA_1 = [<CourseCard cardData={EXAMPLE_CARD_DATA}/>];
+const COURSES_DATA_2 = [<CourseCard cardData={EXAMPLE_CARD_DATA}/>, <CourseCard cardData={EXAMPLE_CARD_DATA}/>];
+
+export default storybook => {
+  return storybook
+    .storiesOf('RecentCourses', module)
+    .addStoryTable([
+      {
+        name: 'Recent Courses - 2 courses ',
+        description: `Recent courses when the teacher has sections enrolled in at least 2 courses.`,
+        story: () => (
+          <RecentCourses
+            courses={COURSES_DATA_2}
+          />
```

Comment:
I suspect we'd be better off passing these as children instead of as a prop, i.e.
```jsx
<RecentCourses>
  <CourseCard cardData={EXAMPLE_CARD_DATA}/>
  <CourseCard cardData={EXAMPLE_CARD_DATA}/>
</RecentCourses>
```

In your component, you can then access these as `this.props.children`